CONTRIBUTING.md

```diff
@@ -1,18 +1,100 @@
+<!-- toc -->
+- [Contributing to Deepchem](#contributing-to-deepchem)
+  - [Getting Started](#getting-started)
+  - [Pull Request Process](#pull-request-process)
+  - [Coding Conventions](#coding-conventions)
+  - [Documentation Conventions](#documentation-conventions)
+- [The Agreement](#the-agreement)
+- [Deepchem Technical Steering Committee](#deepchem-technical-steering-committee)
+<!-- tocstop -->
 
-Once you've got a sense of how the package works, we encourage the use of Github issues to discuss more complex changes, raise requests for new features or propose changes to the global architecture of DeepChem. Once consensus is reached on the issue, please submit a PR with proposed modifications. All contributed code to DeepChem will be reviewed by a member of the DeepChem team, so please make sure your code style and documentation style match our guidelines!
+## Contributing to DeepChem
 
-## Pull Request Process
+We actively encourage community contributions to DeepChem. The first
+place to start getting involved is
+[the tutorials](https://deepchem.readthedocs.io/en/latest/get_started/tutorials.html).
+Afterwards, we encourage contributors to give a shot to improving our documentation.
+While we take effort to provide good docs, there's plenty of room
+for improvement. All docs are hosted on Github, either in `README.md`
+file, or in the `docs/` directory.
 
-Every contribution, must be a pull request and must have adequate time for review by other committers.
+Once you've got a sense of how the package works, we encourage the use
+of Github issues to discuss more complex changes, raise requests for
+new features or propose changes to the global architecture of DeepChem.
+Once consensus is reached on the issue, please submit a PR with proposed
+modifications. All contributed code to DeepChem will be reviewed by a member
+of the DeepChem team, so please make sure your code style and documentation
+style match our guidelines!
 
-A member of the Technical Steering Committee will review the pull request. The default path of every contribution should be to merge. The discussion, review, and merge process should be designed as corrections that move the contribution into the path to merge. Once there are no more corrections, (dissent) changes should merge without further process.
+### Getting Started
+
+To develop DeepChem on your machine, we recommend using Anaconda for managing
+packages. If you want to manage multiple builds of DeepChem, you can make use of
+[conda environments](https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html)
+to maintain seperate Python package environments, each of which can be tied
+to a specific build of DeepChem. Here are some tips to get started:
+
+1. Set up a new conda environment for DeepChem
+
+```bash
+conda create -n deepchem python=3.7
```

Comment:
Let's recommend python 3.8 since that's more standard now

Reply:
@rbharath, rdkit is not compatible with python 3.8. Can we revert to python 3.7?

Reply:
I've used rdkit with py 3.8 (my current dev machine has 3.8 + rdkit) so I think we should be fine hopefully